Libraries/ActionSheetIOS/ActionSheetIOS.js

```diff
@@ -69,10 +69,12 @@ const ActionSheetIOS = {
       destructiveButtonIndices = [destructiveButtonIndex];
     }
 
+    const processedTintColor = processColor(tintColor);
+    invariant(typeof processedTintColor === 'number', 'tintColor must be a simple RGBA value');
```

Comment:
I think saying RGBA can be confusing since it doesn't require `rgba()`. Maybe just `Unexpected color given for tintColor`?

Reply:
ok